CRM/Case/XMLProcessor/Process.php

```diff
@@ -109,14 +109,14 @@ public function process($xml, &$params) {
     foreach ($xml->ActivitySets as $activitySetsXML) {
       foreach ($activitySetsXML->ActivitySet as $activitySetXML) {
         if ($standardTimeline) {
-          if ((boolean ) $activitySetXML->timeline) {
+          if ((bool) $activitySetXML->timeline) {
```

Comment:
Putting an expression inside an `if()` statement already casts it to boolean, so the cast here is redundant and can be removed.